chats/apps/sectorqueue/models.py

```diff
@@ -0,0 +1,100 @@
+from django.db import models
+from chats.core.models import BaseModel
+from django.utils.translation import gettext_lazy as _
+from chats.apps.sectors.models import Sector, SectorAuthorization
+
+
+class SectorQueue(BaseModel):
```

Comment:
Does not need to reference the sector, call it `Queue`. In the same way sectors do not reference projects, it's not `ProjectSector` but `Sector`

Reply:
I change the model name do Queue and remade the migrations, leaving only the initial with the correct name.